### PR TITLE
Returns subsystem weights to previous values

### PR DESCRIPTION
**Pull request by optimumtact**
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights
These are not priority, they are weights

### CODE REVIEW

**Comment by Cyberboss** [comment]
The documentation of these defines really doesn't emphasize that tbh

**Comment by actioninja** [comment]
maybe rename them from priority then

File: code/__DEFINES/subsystems.dm [hunk 1]
```diff
@@ -142,36 +142,37 @@
...
+#define FIRE_PRIORITY_DEFAULT		50
+#define FIRE_PRIORITY_PARALLAX		65
+#define FIRE_PRIORITY_MOBS			100
+#define FIRE_PRIORITY_TGUI			110
+#define FIRE_PRIORITY_TICKER		200
+#define FIRE_PRIORITY_ATMOS_ADJACENCY	300
+#define FIRE_PRIORITY_CHAT			400
+#define FIRE_PRIORITY_RUNECHAT		410
+#define FIRE_PRIORITY_OVERLAYS		500
+#define FIRE_PRIORITY_EXPLOSIONS	666
```
Comment (Dennok) [inline code/__DEFINES/subsystems.dm:172]: All for this?

Comment (Dennok) [inline code/__DEFINES/subsystems.dm:172]: All for this?

**Pull request merged**